spec/gogo/x/cash/codec.proto

```diff
@@ -37,13 +37,16 @@ message FeeInfo {
 }
 
 message Configuration {
-  // TODO: add schema uint32 here
+  weave.Metadata metadata = 1;
+  // Owner is present to implement gconf.OwnedConfig interface
+  // This defines the Address that is allowed to update the Configuration object and is
+  // needed to make use of gconf.NewUpdateConfigurationHandler
```

Comment:
This is gold! :100: 

It took me quite some time to figure this out again, once I forgot that there is a `GetOwner` method expectation that is used by an interface.